### PR TITLE
Make wptrunner chrome always use the WebDriver executor

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Review by jgraham — APPROVED** [review]

File: tools/wptrunner/wptrunner/browsers/chrome.py [hunk 1]
```diff
@@ -30,34 +30,37 @@ def browser_kwargs(test_type, run_info_data, config, **kwargs):
...
+                }
+            },
+            "w3c": True
         }
     }
+
     for (kwarg, capability) in [("binary", "binary"), ("binary_args", "args")]:
         if kwargs[kwarg] is not None:
             capabilities["goog:chromeOptions"][capability] = kwargs[kwarg]
-    if kwargs["headless"]:
```
Comment (jgraham) [inline tools/wptrunner/wptrunner/browsers/chrome.py]: This should have beeen added to `chrome_webdriver`, please [re-]add it before landing.

Comment (gsnedders) [reply tools/wptrunner/wptrunner/browsers/chrome.py]: Added that in #13201, let's land that first

**Comment by gsnedders** [comment]
cc @jugglinmike @Hexcles you guys should know this has landed so you can revert if it breaks wpt.fyi

**Comment by Hexcles** [comment]
@gsnedders thanks for the heads up! This is exciting!

**Pull request merged**